.changeset/bug-tabs.md

```diff
@@ -0,0 +1,5 @@
+---
+'react-magma-dom': minor
```

Comment:
This is a bug fix, so it should go under `patch` instead of minor. 

For reference,
```
Given a version number MAJOR.MINOR.PATCH, increment the:

MAJOR version when you make incompatible API changes
MINOR version when you add functionality in a backwards compatible manner
PATCH version when you make backwards compatible bug fixes
```